### PR TITLE
Add image description best practices

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Proposed changes

Adds basic best practices for adding descriptions to image listing in hub

### CODE REVIEW

File: docker-hub/repos/create.md [hunk 1]
```diff
@@ -32,6 +32,35 @@ When creating a new repository:
   > You can't rename a Docker Hub repository once it's created.
 
 - The description can be up to 100 characters. It is used in the search results.
+
+  <div class="panel panel-default">
+    <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapseSample1" style="cursor: pointer">
+    Description best practices
+    <i class="chevron fa fa-fw"></i></div>
+    <div class="collapse block" id="collapseSample1">
+      <p>A good image description is essential to help potential users understand why and how to use the image. The bellow covers the best practices to follow when adding a description to your image.</p>
```
Comment (stephaurelio) [inline docker-hub/repos/create.md:41]: Small typo: 
```suggestion
      <p>A good image description is essential to help potential users understand why and how to use the image. The below covers the best practices to follow when adding a description to your image.</p>
```
or to stay consistent with style guide:
```suggestion
      <p>A good image description is essential to help potential users understand why and how to use the image. The following covers the best practices to follow when adding a description to your image.</p>
```

File: docker-hub/repos/create.md [hunk 2]
```diff
@@ -32,6 +32,35 @@ When creating a new repository:
...
+      <ul>
+      <li>Exposing ports</li>
+      <li>Default environment variables</li>
+      <li>Non-root user</li>
+      <li>Running in debug mode</li>
+      </ul>
+      <h3>Image tags and variants</h3>
+      <p>List the key image variants and the tags for using them along with what that variant offers and why someone might want to use that variant.</p>
+      <h3>Where to find more information</h3>
+      <p>Add links here for docs and support sites, communities, or mailing lists were users can find more and ask questions.</p>
```
Comment (stephaurelio) [inline docker-hub/repos/create.md:57]: Small typo:
```suggestion
      <p>Add links here for docs and support sites, communities, or mailing lists where users can find more and ask questions.</p>
```

File: docker-hub/repos/create.md [hunk 3]
```diff
@@ -32,6 +32,35 @@ When creating a new repository:
...
+      <h3>Who the image is maintained by</h3>
```
Comment (stephaurelio) [inline docker-hub/repos/create.md:58]: Suggestion to change to avoid dangling preposition:
```suggestion
      <h3>Who maintains the image</h3>
```

**Review by stephaurelio — COMMENTED** [review]
Left a few comments for small updates!

File: docker-hub/repos/create.md (hunk 2 above)
Comment (ChrisChinchilla) [reply docker-hub/repos/create.md:57]: @stephaurelio Haha, I missed this in VS Code as it was HTML and not markdown, but clearly Notion has a terrible spell checker too 😆

File: docker-hub/repos/create.md (hunk 3 above)
Comment (ChrisChinchilla) [reply docker-hub/repos/create.md:58]: @stephaurelio These were worded intentionally slightly differently as we are instructing someone else to write docs, but let me have a think about it.

**Comment by ChrisChinchilla** [comment]
@stephaurelio If you feel like another look, updated copy a bit and actually moved things around as we realised the user experience creating a repo is different.

File: docker-hub/repos/create.md [hunk 4]
```diff
@@ -41,6 +41,39 @@ When creating a new repository:
...
+### Add a repository overview
+
+Once you have created a repository, add an overview to the **Repository overview** field. This describes what your image does and how to use it.
+
+<div class="panel panel-default">
+  <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapseSample1" style="cursor: pointer">
+  Repository overview best practices
+  <i class="chevron fa fa-fw"></i></div>
+  <div class="collapse block" id="collapseSample1">
+    <p>A good image description is essential to help potential users understand why and how to use the image. The bellow covers the best practices to follow when adding a description to your image.</p>
```
Comment (stephaurelio) [inline docker-hub/repos/create.md:53]: Looks like the typos snuck back in, possibly when the content moved:

```suggestion
    <p>A good image description is essential to help potential users understand why and how to use the image. The below covers the best practices to follow when adding a description to your image.</p>
```

File: docker-hub/repos/create.md [hunk 5]
```diff
@@ -41,6 +41,39 @@ When creating a new repository:
...
+    <ul>
+    <li>Exposing ports</li>
+    <li>Default environment variables</li>
+    <li>Non-root user</li>
+    <li>Running in debug mode</li>
+    </ul>
+    <h3>Image tags and variants</h3>
+    <p>List the key image variants and the tags for using them along with what that variant offers and why someone might want to use that variant.</p>
+    <h3>Where to find more information</h3>
+    <p>Add links here for docs and support sites, communities, or mailing lists were users can find more and ask questions.</p>
```
Comment (stephaurelio) [inline docker-hub/repos/create.md:69]:
```suggestion
    <p>Add links here for docs and support sites, communities, or mailing lists where users can find more and ask questions.</p>
```

**Review by stephaurelio — COMMENTED** [review]
Looks good! Not sure what happened but looks like the commits from earlier were overrode, so just added those again as suggestions.

File: docker-hub/repos/create.md [hunk 6]
```diff
@@ -41,6 +41,39 @@ When creating a new repository:
...
+
+Once you have created a repository, add an overview to the **Repository overview** field. This describes what your image does and how to use it.
+
+<div class="panel panel-default">
+  <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapseSample1" style="cursor: pointer">
+  Repository overview best practices
+  <i class="chevron fa fa-fw"></i></div>
+  <div class="collapse block" id="collapseSample1">
+    <p>A good image description is essential to help potential users understand why and how to use the image. The below covers the best practices to follow when adding a description to your image.</p>
+    <h3>Describe the image</h3>
```
Comment (aevesdocker) [inline docker-hub/repos/create.md:54]: I'd make these h4. 

From the TOC it looks like they're all on one level, and when the accordion is closed I'm expecting to see these headers but don't.

image

**Comment by ChrisChinchilla** [comment]
@aevesdocker Not at all, actually I don't want those headings in the toc at all, so fixed that.

**Review by aevesdocker — APPROVED** [review]
Nice one. LGTM 👍

**Pull request merged**